lib/ueberauth/strategy/cognito/config.ex

```diff
@@ -14,14 +14,18 @@ defmodule Ueberauth.Strategy.Cognito.Config do
     :jwt_verifier
   ]
 
+  @scope_keys [
+    :scope
+  ]
+
   @optional_keys [
     :uid_field,
     :name_field
   ]
 
   @enforce_keys @strategy_keys ++ @dependency_keys
 
-  defstruct @enforce_keys ++ @optional_keys
+  defstruct @enforce_keys ++ @optional_keys ++ @scope_keys
```

Comment:
Can `:scope` be one of the `@optional_keys`? I think that would work, and you could maintain the default in `Ueberauth.Strategy.Cognito` via `scope: scope || "openid profile email"`.

Reply:
That should work, updated accordingly!